### PR TITLE
Add inspection for references to Mixin classes

### PR DESCRIPTION
**Pull request by Aaron1011**

### CODE REVIEW

File: src/main/java/com/demonwav/mcdev/platform/mixin/inspections/MixinClassReferenceInspection.java [hunk 1]
```diff
@@ -0,0 +1,64 @@
+package com.demonwav.mcdev.platform.mixin.inspections;
```
Comment (kashike) [inline src/main/java/com/demonwav/mcdev/platform/mixin/inspections/MixinClassReferenceInspection.java]: Missing licence leader.

**Comment by JBYoshi** [comment]
Would this skip cases such as one mixin extending another where the reference gets replaced at runtime?

**Comment by DemonWav** [comment]
This isn't mergeable as it is, @JBYoshi, I've told @Aaron1011 what needs to be changed for it to work right. Currently it doesn't check for extends or implements, but it will when it's pulled.

**Comment by DemonWav** [comment]
Merged.

**Pull request closed without merging**